tests/unit_tests/test_metagenome.py

```diff
@@ -162,7 +164,41 @@ def test_orfs_called(metagenome, monkeypatch):
     with monkeypatch.context():
         # Here we set the filepaths to the metagenome fixture assembly path
         # (since we know this exists and is non-empty)
+        # metagenome.__str__ == metagenome.assembly
         monkeypatch.setattr(metagenome, "prot_orfs_fpath", str(metagenome))
         monkeypatch.setattr(metagenome, "nucl_orfs_fpath", str(metagenome))
```

Comment:
This is more explicit. I should have aliased the context in the first place

```suggestion
    with monkeypatch.context() as m:
        # Here we set the filepaths to the metagenome fixture assembly path
        # (since we know this exists and is non-empty)
        # metagenome.__str__ == metagenome.assembly
        m.setattr(metagenome, "prot_orfs_fpath", str(metagenome))
        m.setattr(metagenome, "nucl_orfs_fpath", str(metagenome))
```